intake/catalog/base.py

```diff
@@ -298,26 +302,36 @@ def __init__(self, url, http_args={}, page_size=None, **kwargs):
         page_size : int, optional
             The number of entries fetched at a time during iteration.
             Default is None (no pagination; fetch all entries in bulk).
+        query : Iterable or None
+            List from a sequence of progressive searches. Each element is
+            expected to be a valid argument to Catalog.search() for this
+            Catalog.
         kwargs: may include catalog name, metadata, source ID (if known) and
             auth instance.
         """
+        secure = http_args.pop('ssl', False)
```

Comment:
See commit message on https://github.com/ContinuumIO/intake/pull/208/commits/5a66628bead4eaa7910db3f7ed5dad5ee10af84a for description of why this change is scope for this PR.